dags/vz_afd_ems_import.py

```diff
@@ -1,82 +1,57 @@
-# stuff to make the airflow, 1Password integration work
 import os
 import pendulum
-from airflow.decorators import dag, task
+from airflow.decorators import dag
+from airflow.operators.docker_operator import DockerOperator
+from utils.slack_operator import task_fail_slack_alert
 
-DEPLOYMENT_ENVIRONMENT = os.environ.get("ENVIRONMENT", 'development')   # our current environment from ['production', 'development']
 
-ENVIRONMENT = {
```

Comment:
This caught my eye at first, but I see why this is removed since it is happening again in the script that is run. 👍

Reply:
Thanks for observing that! The airflow environment has the 3 1PW related keys and the deployment class (production or development), and the scripts look for those values in the same environment variable names, so I can just pass the whole environment down into the container with `environment=dict(os.environ),` in the `DockerOperator`.